manifests/init.pp

```diff
@@ -15,11 +15,16 @@
   $user                 = $::caddy::params::user,
   $group                = $::caddy::params::group,
   $install_method       = $::caddy::params::install_method,
-  $release_file_name    = $::caddy::params::release_file_name,
```

Comment:
@iuriaranda checked SAP and Viafutura. No references to these parameters so I think we are safe.